deps/rabbitmq_stream/src/rabbit_stream_reader.erl

```diff
@@ -366,19 +381,23 @@ listen_loop_pre_auth(Transport,
                                           Connection3,
                                           State1,
                                           Configuration);
-                failure ->
-                    close(Transport, S, State);
                 _ ->
-                    listen_loop_pre_auth(Transport,
-                                         Connection1,
-                                         State1,
-                                         Configuration)
+                    rabbit_log:warning("Closing socket ~w because transition from ~p to ~p is not allowed",
```

Comment:
This should probably use`rabbit_log_connection` so that the message is categorized.

Reply:
Thanks. I changed all connection related log messages in this file from `rabbit_log` to `rabbit_log_connection`.